lib/sda/host.go

```diff
@@ -342,8 +352,8 @@ func (h *Host) processServiceMessage(e *network.Entity, m *ServiceMessage) {
 		// 404 Service Unknown
 		return
 	}
-	dbg.Lvl3("host", h.Address(), " => Dispatch request to ServiceMessage")
-	s.ProcessServiceMessage(e, m)
+	dbg.Lvl5("host", h.Address(), m)
+	go s.ProcessServiceMessage(e, m)
```

Comment:
Because a serviceMessageHandler might produce another serviceMessage and wait for its return before continuing.